plugin/trino-ignite/src/main/java/io/trino/plugin/ignite/IgniteClient.java

```diff
@@ -193,7 +193,20 @@ public IgniteClient(
     @Override
     public Collection<String> listSchemas(Connection connection)
     {
-        return ImmutableSet.of(IGNITE_SCHEMA);
+        try (ResultSet resultSet = connection.getMetaData().getSchemas()) {
+            ImmutableSet.Builder<String> schemaNames = ImmutableSet.builder();
+            while (resultSet.next()) {
+                String schemaName = resultSet.getString("TABLE_SCHEM");
+                // skip internal schemas
+                if (filterSchema(schemaName)) {
+                    schemaNames.add(schemaName);
+                }
+            }
+            return schemaNames.build();
+        }
+        catch (SQLException e) {
+            throw new TrinoException(JDBC_ERROR, "Failed to list schemas", e);
+        }
```

Comment:
`IgniteClient.getTables` method still has the following comment:
```
// TODO: https://github.com/trinodb/trino/issues/8552 support user custom schemas.
```

Could you update the method as well?

Reply:
Ok